pyTigerGraph/pyTigerGraphBase.py

```diff
@@ -257,20 +317,27 @@ def _req(self, method: str, url: str, authMode: str = "token", headers: dict = N
         if logger.level == logging.DEBUG:
             logger.debug("params: " + self._locals(locals()))
 
-        if authMode == "token" and str(self.apiToken) != "":
-            if isinstance(self.apiToken, tuple):
-                self.apiToken = self.apiToken[0]
-            self.authHeader = {'Authorization': "Bearer " + self.apiToken}
-            _headers = self.authHeader
-        else:
-            self.authHeader = {'Authorization': 'Basic {0}'.format(self.base64_credential)}
-            _headers = self.authHeader
-            authMode = 'pwd'
+        # If JWT token is provided, always use jwtToken as token
```

Comment:
Is jwtToken also used when authMode is pwd?

Reply:
currently no, still use pwd. in the init, if the db doesn't support jwt, it will raise an error asking users to use username and password.